# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/ipc/GpuDecoderModule.h
# Commit: 1932f2f34993
# Full Hash: 1932f2f3499302f712bb680096d51478e53c99ed
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2020-10-21 09:51:28
# Regressor Bug: 1595994
# File Overlap Count: 1
# Description:
#   Bug 1595994 - P6: Change Supports to take SupportDecoderParams. r=kamidphish
#   
#   This is a subset of the parameters passed via CreateDecoderParams and is so
#   Supports() calls have access to KnowsCompositor and Options when determining
#   if decoding is supported.
# ==============================================================================

diff -r 0606794a9e8c -r 1932f2f34993 dom/media/ipc/GpuDecoderModule.h
--- a/dom/media/ipc/GpuDecoderModule.h	Tue Oct 20 23:24:48 2020 +0000
+++ b/dom/media/ipc/GpuDecoderModule.h	Tue Oct 20 23:26:25 2020 +0000
@@ -27,7 +27,7 @@
 
   bool SupportsMimeType(const nsACString& aMimeType,
                         DecoderDoctorDiagnostics* aDiagnostics) const override;
-  bool Supports(const TrackInfo& aTrackInfo,
+  bool Supports(const SupportDecoderParams& aParams,
                 DecoderDoctorDiagnostics* aDiagnostics) const override;
 
   already_AddRefed<MediaDataDecoder> CreateVideoDecoder(